Extension/src/LanguageServer/ui.ts

```diff
@@ -168,6 +168,25 @@ export class UI {
             });
     }
 
+    public showCompileCommands(paths: string[]): Thenable<number> {
+        let options: vscode.QuickPickOptions = {};
+        options.placeHolder = "Select a compile_commands.json...";
+
+        let items: IndexableQuickPickItem[] = [];
+        for (let i: number = 0; i < paths.length; i++) {
+            items.push({label: paths[i], description: "", index: i});
+        }
+        items.push({ label: "Edit Configurations...", description: "", index: paths.length });
```

Comment:
I didn't see this in the spec.  You should ask Rong if she wants this or not.  My vote is that it's unnecessary.